lib/ts/controllers/s-popover.ts

```diff
@@ -194,6 +194,15 @@ namespace Stacks {
         private boundHideOnOutsideClick!: any;
         private boundHideOnEscapePress!: any;
 
+        connect() {
+            super.connect();
+
+            if (this.data.get("show-on-connect") === "true") {
```

Comment:
I'd move this into the `BasePopoverController` and drop the `connect` function here. It'll likely never be used on tooltip, but it gives us the option and will work for other future inheritors

Reply:
That's reasonable.  Will do.